src/styles/index.scss

```diff
@@ -1,3 +1,44 @@
+@import './units/variables';
+@import './blocks/button-buy';
+@import './blocks/stars';
+@import './blocks/card-positions';
+@import './blocks/price';
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+}
+
 body {
   margin: 0;
+  font-size: 12px;
+}
```

Comment:
You have defined `font-size: 12px;` twice, in both `html` and `body` selectors. It's redundant and can be confusing. Define it once in the `html` selector and it will apply to the `body` as well.